tools/run-fbp-tests

```diff
@@ -269,6 +270,7 @@ def run_tests(args):
                 logging.warning("Dependence test %s for test %s failed. Skipping it",
                         curr.depends, curr.test_file)
                 status["skipped"] += 1
+                logging.info("FAIL: %s" % path["file"])
```

Comment:
I've read your comment on the other PR (to ".bb"), but I'll reply it here: this is not FAIL, it's `SKIP` or just omit it. You didn't test, so there is no way to know if this failed or not.